controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorGroupImpl.java

```diff
@@ -82,7 +82,13 @@
 
     void initialize() throws CheckpointStoreException {
```

Comment:
there isnt a simple test that i can author here because there is no existing event processor group unit test where i could call initialize twice.. 
i can add a test where i mock most of the things. 
And controllerEventProcessor should ensure its initialize is also idempotent (if its not testing it already)